x-pack/plugin/esql/qa/testFixtures/src/main/resources/enrich.csv-spec

```diff
@@ -645,7 +645,7 @@ IDR       |  Indore       |  POINT(75.8472 22.7167)  |  India      |  POINT(75.8
 ;
 
 enrichDroppingOriginalIndexFields
-required_capability: enrich_load
+required_capability: fix_field_names_enrich_lookup_no_valid_fields
```

Comment:
I think this _also_ needs `enrich_load`. This one prevents from running in CsvTests, too.